src/components/Groups/StationsTab.vue

```diff
@@ -343,16 +374,24 @@ export default {
       )
         ? "thermal camera"
         : "camera";
-      this.pendingStations = monitoring
+      const csvCameras = monitoring
         .filter((item) => item.Type === cameraKey)
         .map((item) => ({
           name: item["Number / Code"],
           latitude: item.Lat,
           longitude: item.Lon,
         }));
+      if (!csvCameras.length) {
+        this.invalidCsvFormat = true;
+        this.invalidCsvReason =
+          "Supplied CSV has no columns labeled either 'Camera' or 'Thermal Camera'";
```

Comment:
This is a bit confusing.   I think you mean 
"Supplied CSV has no rows where the 'type' is 'camera' or 'Thermal camera'"

It's the type column not the other way around?